basis/ui/backend/gtk/gtk.factor

```diff
@@ -420,9 +425,11 @@ M: window-handle flush-gl-context ( handle -- )
 
 M:: gtk-ui-backend (open-window) ( world -- )
     GTK_WINDOW_TOPLEVEL gtk_window_new :> win
+    gtk_drawing_area_new :> drawable
+    win drawable gtk_container_add
```

Comment:
So the main idea is we have a top level container that we add everything into?  Whereas before we used the window as the top level?